shopinvader/services/settings.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2021 Akretion (http://www.akretion.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.base_rest import restapi
+from odoo.addons.component.core import Component
+
+
+class ExportSettingsService(Component):
+    """Shopinvader service to export allowed settings"""
+
+    _inherit = [
+        "base.shopinvader.service",
+    ]
+    _name = "shopinvader.export.settings.service"
+    _usage = "settings"
+    _description = __doc__
+
+    def get(self):
+        return {
+            **self._get_country(),
+            **self._get_title(),
+            **self._get_industry(),
+            **self._get_currency(),
+            **self._get_lang(),
+        }
+
+    def _jsonify_fields_country(self):
+        return [
+            "name",
+            "code",
+            "id",
+        ]
+
+    def _get_country(self):
+        countries = self.shopinvader_backend.allowed_country_ids.jsonify(
+            self._jsonify_fields_country()
+        )
+        return {"countries": countries}
+
+    @restapi.method(
+        [(["/country/get"], "GET")],
+        auth="api_key",
+    )
+    def get_country(self):
```

Comment:
```suggestion
    def countries(self):
```
aaand repeat for all endpoint IMO :wink: